content/user/admin_pages/catalog/attributes_controller.md

```diff
@@ -123,15 +123,15 @@ As you can see, the number you enter is the maximum limit of the discount, and t
 
 Negative numbers work for in these entries as well, making them quantity discounts or quantity charges. This is useful if you define a base price and would like to allow the user to have the discount off the base price.
 
-## Attribute Flags (the coloured boxes)
+## Attribute Flags (the colored boxes)
```

Comment:
but ... but ... ! :D

Reply:
LOL I can take out if you wish.  I keep forgetting you are not here in 'Murica.

Reply:
Nah, I'm just poking fun.